lib/qos-packet.js

```diff
@@ -10,7 +10,7 @@ function QoSPacket (original, client) {
 
   if (!original.messageId) {
     this.messageId = client._nextId
-    if (client._nextId === 65535) {
+    if (client._nextId >= 65535) {
```

Comment:
why? could this happen?

Reply:
depends, it prevents someone set `_nextId` by themselves.